source/templates.tex

```diff
@@ -1390,16 +1388,14 @@
   T& elem(int i) { return v[i]; }
 };
 \end{codeblock}
-
-\pnum
 The prefix
 \tcode{template}
 \tcode{<class}
 \tcode{T>}
 specifies that a template is being declared and that a
 \grammarterm{type-name}
 \tcode{T}
-will be used in the declaration.
+may be used in the definition.
```

Comment:
OK, you're right, I'll back those changes out for now. It seems like the status quo is fine, and this isn't making it better.